libraries/lib-project-rate/ProjectRate.cpp

```diff
@@ -87,9 +88,8 @@ static ProjectFileIORegistry::WriterEntry entry {
 static ProjectFileIORegistry::AttributeReaderEntries entries {
 // Just a pointer to function, but needing overload resolution as non-const:
 (ProjectRate& (*)(AudacityProject &)) &ProjectRate::Get, {
-   { L"rate", [](auto &settings, auto value){
-      double rate;
-      Internat::CompatibleToDouble(value, &rate);
+   { "rate", [](auto &settings, auto value){
+      double rate = value.Get(settings.GetRate());
```

Comment:
Squash the commit that made this change with the previous, to keep buildability of all commits in the branch

Reply:
I will squash it before merging